tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 1, len(clusterList.Data))
 
-	err = users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner, nil)
+	err = users.AddProjectMember(standardUserClient, clusterOwnerProject, additionalUser, roleProjectOwner, nil)
 	require.NoError(rb.T(), err)
-	userGetProject, err := projects.GetProjectList(rb.additionalUserClient, rb.cluster.ID)
+	projectListProjectOwner, err := projects.GetProjectList(additionalUserClient, rb.cluster.ID)
 	require.NoError(rb.T(), err)
-	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
-
+	assert.Equal(rb.T(), clusterOwnerProject.Name, projectListProjectOwner.Data[0].Name)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateAddPOsAsProjectOwner() {
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+func (rb *RBACAdditionalTestSuite) testSetupUserAndProject() (*rancher.Client, *management.Project) {
+	rb.T().Log("Set up User with cluster Role for additional rbac test cases " + roleOwner)
+	newUser, standardUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	rb.standardUserCOProject = createProjectAsCO
 
-	log.Info("Validating if Project Owner can add another Project Owner")
-	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner, nil)
-	require.NoError(rb.T(), errUserRole)
-	rb.additionalUserClient, err = rb.additionalUserClient.ReLogin()
+	createProjectAsAdmin, err := createProject(rb.client, rb.cluster.ID)
 	require.NoError(rb.T(), err)
 
-	addNewUserAsPO, err := users.CreateUserWithRole(rb.client, users.UserConfig(), standardUser)
-	require.NoError(rb.T(), err)
-	addNewUserAsPOClient, err := rb.client.AsUser(addNewUserAsPO)
+	log.Info("Adding a standard user as project Owner in the admin project")
+	errUserRole := users.AddProjectMember(rb.client, createProjectAsAdmin, newUser, roleProjectOwner, nil)
+	require.NoError(rb.T(), errUserRole)
+	standardUserClient, err = standardUserClient.ReLogin()
 	require.NoError(rb.T(), err)
+	return standardUserClient, createProjectAsAdmin
+}
 
-	errUserRole2 := users.AddProjectMember(rb.additionalUserClient, rb.standardUserCOProject, addNewUserAsPO, roleProjectOwner, nil)
-	require.NoError(rb.T(), errUserRole2)
+func (rb *RBACAdditionalTestSuite) TestProjectOwnerCanAddProjectOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	addNewUserAsPOClient, err = addNewUserAsPOClient.ReLogin()
+	standardUserClient, adminProject := rb.testSetupUserAndProject()
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
 
-	userGetProject, err := projects.GetProjectList(addNewUserAsPOClient, rb.cluster.ID)
+	log.Info("Validating if Project Owner can add another Project Owner")
```

Comment:
nit; I don't think the logs in this function are helpful / they seem redundant.

Reply:
Each of these tests have multiple validations. I think logs might make it more easier to understand each of these validations.